manager/src/main/java/com/redhat/service/bridge/manager/models/Bridge.java

```diff
@@ -55,7 +52,7 @@
 
     @Column(name = "status")
     @Enumerated(EnumType.STRING)
-    private BridgeStatus status;
+    public BridgeStatus status;
```

Comment:
Why this change?

Reply:
It's a leftover of a wrong automatic refactoring by IntelliJ 😅

Reply:
Fixed :+1: